docs/changes/CHANGES

```diff
@@ -37,6 +72,9 @@ Missing features from 1.9.x
 * SVG editor
 * LDAP may have missing dependencies depending upon OS flavor
 * WikiDicts have limited function, does not use Babel translations
+* language support limited to English, Deutsch, Portuguese (Brasil), Russian
```

Comment:
Maybe rather "German".

Guess whoever does not understand English won't read the changelog anyway.